docs/docs/1-getting-started/1-contributors/2-running-tests.mdx

```diff
@@ -0,0 +1,46 @@
+## **Running Tests**
+
+Once your local Consul Agent is running, you can verify your changes and contributions by running the tests.
+
+---
+
+**PS**: **Make sure you’ve written or updated unit tests relevant to your contribution.**
+### Option 1: Using Visual Studio's GUI.
+
+If you’re using Visual Studio:
+
+1. Open the **Test Explorer** panel (`Test` → `Test Explorer`).
+2. Build the project to ensure all test assemblies are loaded.
+3. Select the test or test class you want to run.  
+   You can filter by project or namespace if needed.
+4. Click **Run** or **Debug** to execute the selected test(s).
+
+
+---
+
+### Option 2: Using the Command Line
+
+If you prefer using the command line:
+
+1. Navigate into the `Consul.Test` project directory.
+   ```bash
+   cd Consul.Test
+   ```
+2. Run the tests in the Consul.Test project using the following command:
+    ```bash
+    dotnet test
+    ```
+    This command will build the test project and execute all tests, providing output in the terminal that looks like this:
```

Comment:
nit: Using the present tense ("builds," "executes") would make the description of the command's action more immediate and direct.